### ISSUE TITLE
France - tv-programe.telecablesat.fr

### ISSUE THREAD

**Issue by Robinmcgvr** [Robinmcgvr/Kapow-Data#202]
Downloads from this site have now ceased as of 17/07
I've got Matteo looking into it ...

**Comment by GoodallE** [comment]
On 2023-07-24 12:30, Robinmcgvr wrote:
> Downloads from this site have now ceased as of 17/07
> I've got Matteo looking into it ...
Thanks Robin ;)

**Comment by Robinmcgvr** [comment]
The site is surrently blocking us - Matteo has started a work around, which will make the data for this site an extra day behind if it works.
We may also need to drop some channels, I ahve identified ones we don't need, such as belgian german swiss and italian ones this source picks up at the moment.
I am hoping we can keep all the french ones in case they are needed at a later date.

**Comment by Lynne12** [comment]
Website returns on 31 July, but none of the channels required for online scheduling are available for loading.

**Comment by Robinmcgvr** [comment]
Yes this is still a work in progress - we are seeing if we can get it to work fully again, I will update at the end of this week.

**Comment by Robinmcgvr** [comment]
More on this data - for all online scheduling channels (Cinc. Cinf. serieclub , Paris Premiere and Teva) the data stopped giving us the full day from 14/07 onwards
All these channels are available via programm-tv.net - this site though does not give episode information
Paris Premiere and Teva are also available via programme-television.org
Cinc and cinf are currently also available via Belgian site cinetelerevue
So we can cover all the channels apart from Serieclub from other sites we already download at the moment should we need to abandon telcablesat as a loading source
I am seein gif I can find sericlub somewhere else

**Comment by Robinmcgvr** [comment]
I have saved data for Channel Sericlub from the TVEpg.eu site for channel Serieclub as this seems to be the only one we don't have an alternative source for .
It will be a pain if we end up having to plug this gap manually, but it might come to it.

**Comment by Lynne12** [comment]
@Robinmcgvr what is happening with this, please?  I am loading 29 Aug-4 Sep week and this shows 4 September missing.

**Comment by Robinmcgvr** [comment]
is that a specific channel that is missing?

**Comment by Robinmcgvr** [comment]
There is a lag on the data from this source now due to the way Matteo has to treat it to get around the security, so it will be a few days behind -

**Comment by Robinmcgvr** [comment]
Anyway thanks for the heads up on the 4th being missing , which i can see it is now, as this would not have been picked up til tomorrow, the robots are only checked twice a week
and note that on a monday we see data up to sunday and friday up to thursday in most cases

**Comment by Robinmcgvr** [comment]
Did you switch your loading source for any of the channels ?
See August 3 entry above for alternative sources mentioned for some of these

**Comment by Lynne12** [comment]
I haven't done anything, as you have not given any specific instructions.  Most of the comments appear to be note gathering.  If you would like loading sources switched, please advise which ones and the websites we will now use.  The less websites the better, as it causes more work.

**Comment by Robinmcgvr** [comment]
what was the last period loaded for the french data?

**Comment by Robinmcgvr** [comment]
sorry seen that now

**Comment by Lynne12** [comment]
July is missing 14th-31st
August is missing 1st

**Comment by Robinmcgvr** [comment]
@Lynne12 
so we can load the gaps from the following sources:
programme-television.org (this site includes the episode info in a way that should import into studio - so the better source to use, sorry)
Teva, Paris Premiere, Syfy, 

programme-tv.net
cinc and cinf (movies so don't need ep info)

Serieclub only has an alternative source for 06/08/2023 onwards, so unfortunately I am going to be stuck with manually scheduling that which is a pain

**Comment by Robinmcgvr** [comment]
n.b. actually need to go back to the 14/07 as we only have the very early hours for that date - guess we will get overlap potentially for two records per day for that period

**Comment by Lynne12** [comment]
@Robinmcgvr I have asked Richard to set up the alternate websites, so that I can load the missing data.  Will wait to hear when the serieclub 1 August is manually scheduled.

**Comment by Lynne12** [comment]
> n.b. actually need to go back to the 14/07 as we only have the very early hours for that date - guess we will get overlap potentially for two records per day for that period

I've deleted the overlaps.

**Comment by Lynne12** [comment]
> @Lynne12 so we can load the gaps from the following sources: programme-television.org (this site includes the episode info in a way that should import into studio - so the better source to use, sorry) Teva, Paris Premiere, Syfy,
> 
> programme-tv.net cinc and cinf (movies so don't need ep info)
> 
> Serieclub only has an alternative source for 06/08/2023 onwards, so unfortunately I am going to be stuck with manually scheduling that which is a pain

@Robinmcgvr Serieclub only needs 1 August manually scheduled.  The remainder of August data is loaded for online scheduling.  Can you please let me know which website I should be using to load Serieclub going forwards?

**Comment by Lynne12** [comment]
@Robinmcgvr just reading back on this.  Are we no longer using telecablesat website, going forward, for any channels?  If so I need to set up the gap filling websites (as per your messages above), as the permanent loading websites.

**Comment by Robinmcgvr** [comment]
re, Serie club we should use the site programme-television.org

**Comment by Robinmcgvr** [comment]
I think it would probably be good idea to use the above source for all of the loading going forwards as it carries all the channels and it would keep things simple to use a single source

**Comment by Lynne12** [comment]
@Robinmcgvr Loading source changed for all 6 channels.  Just 1st August for you to manually schedule for Serieclub.

**Comment by Robinmcgvr** [comment]
@Lynne12 
@Ma-rco 
We are missing 13/01/2024 for this one

**Comment by Lynne12** [comment]
@Robinmcgvr can this be closed, please?  We can keep the main French issue open.
